.gitignore

```
@@ -1 +1 @@
-/target
+/target
```

Should keep the newline
```suggestion
/target

```